ddht/v5_1/rpc_handlers.py

```diff
@@ -51,48 +80,47 @@ def validate_endpoint(value: Any) -> None:
         raise RPCError(f"Invalid Endpoint: {value}")
 
 
-class PingHandler(RPCHandler[Tuple[NodeID, Optional[Endpoint]], PongResponse]):
-    def __init__(self, network: NetworkAPI) -> None:
-        self._network = network
+def validate_and_extract_destination(value: Any) -> Tuple[NodeID, Optional[Endpoint]]:
+    node_id: NodeID
+    endpoint: Optional[Endpoint]
 
-    def extract_params(self, request: RPCRequest) -> Tuple[NodeID, Optional[Endpoint]]:
-        try:
-            raw_params = request["params"]
-        except KeyError as err:
-            raise RPCError(f"Missiing call params: {err}")
+    if is_hex_node_id(value):
+        node_id = NodeID(decode_hex(value))
+        endpoint = None
+    elif value.startswith("enode://"):
```

Comment:
I don't think there are any tests which cover this branch and then try to send a `FindNode` to the provided endpoint

Reply:
It's covered under the parametrized fixture which enumerates the different supported formats (I verified by dropping an `assert False` into this context block and seeing the failure)